tests/trading/test_wcl_fuzzy.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from decimal import *
```

Comment:
Usually we're explicit. I don't mind this that much but should probably be consistent

Reply:
Fixed.